spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue2436Test.java

```diff
@@ -0,0 +1,24 @@
+package edu.umd.cs.findbugs.detect;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.annotations.Confidence;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+import org.junit.Test;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class Issue2436Test extends AbstractIntegrationTest {
+
+    @Test
+    public void test() {
+        performAnalysis("ghIssues/Issue2436.class");
+        BugInstanceMatcher bugMatcher = new BugInstanceMatcherBuilder()
+                .bugType("SE_NO_SERIALVERSIONID")
+                .withConfidence(Confidence.LOW)
+                .build();
+        assertThat(getBugCollection(), containsExactly(1, bugMatcher));
+    }
+
+    }
```

Comment:
Can you please modify the indenting to be conform to the project? Or you can simply run `./gradlew spotlessApply` or more specifically the `./gradlew :spotbugs-tests:spotlessApply` command as you can see in the build fail.